### PR TITLE
Rdfa attributes rework

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by BjornTwachtmann** [comment]
Looks good, merge it.

File: src/ValueObject/MetaContext.php [hunk 1]
```diff
@@ -75,11 +68,21 @@ public function canonicalUrl(): string
         return $this->canonicalUrl;
     }
 
+    public function context(): mixed
```
Comment (votemike) [inline src/ValueObject/MetaContext.php]: Didn't know you could use `: mixed`

File: src/ValueObject/MetaContext.php [hunk 2]
```diff
@@ -29,22 +25,19 @@ class MetaContext
...
-    /** @var string */
-    private $schemaType = '';
-
     /** @var bool */
     private $showAdverts = false;
 
     public function __construct($context = null, string $canonicalUrl = '')
     {
         $this->canonicalUrl = $canonicalUrl;
+        $this->context = $context;
```
Comment (votemike) [inline src/ValueObject/MetaContext.php]: Do you need to initialise this property?

File: src/ValueObject/MetaContext.php (hunk 1 above)
Comment (felipeparaujo) [inline src/ValueObject/MetaContext.php]: It's useless having a mixed return type tho, it's probably the default value

File: src/ValueObject/MetaContext.php (hunk 2 above)
Comment (felipeparaujo) [inline src/ValueObject/MetaContext.php]: yes, I need that to get the schema type

Comment (votemike) [inline src/ValueObject/MetaContext.php]: I mean do you need:
```
/** @var CoreEntity|Network|Etc... */
private $context;
```

Comment (felipeparaujo) [inline src/ValueObject/MetaContext.php]: oh yeah forgot about that lol

File: src/ValueObject/MetaContext.php (hunk 1 above)
Comment (BPScott) [inline src/ValueObject/MetaContext.php]: I don't think it's got much value

**Pull request merged**